src/vtk_calculator.py

```diff
@@ -44,7 +44,8 @@ def parse_args():
                         help="The mesh (VTK Unstructured Grid) used as input")
     parser.add_argument("--function", "-f", dest="function", required=True,
                         help="""The function to evalutate on the mesh.
-            Syntax is the same as used in the calculator object, coordinates are given as e.g.  'cos(x)+y'.""")
+            Syntax is the same as used in the calculator object, coordinates are given as e.g.  'cos(x)+y'.
+            There also some predefined functions exist 'franke' 'eggholder' 'rosenbrock' """)
```

Comment:
```suggestion
            Alternatively, specify one of the following predefined functions: 'franke' 'eggholder' 'rosenbrock' """)
```

Also, please check how this formatting looks in the console. `vtk_calculator.py --help`

Reply:
Thanks for the suggestion.